core/src/banking_stage.rs

```diff
@@ -1620,4 +1694,59 @@ mod tests {
             })
             .collect_vec();
     }
+
+    #[test]
```

Comment:
Can we also add a test that tests the pipeline of stashing deserialized packets where:

1) `process_received_packets` adds a mixture of deserialized and serialized packets that get added to the buffer
2) the `consume_or_forward_packets`function deserializes some of the serialized packets
3) Ensure that both types of packets can be successfully processed next time the buffer is consumed by "consume_buffered_packets"